automation_tools/__init__.py

```diff
@@ -388,17 +388,13 @@ def setup_firewall(definitions=None, flush=True):
         if run('systemctl status firewalld', quiet=True).failed:
             run('systemctl enable firewalld')
             run('systemctl start firewalld')
-        exists_command = 'firewall-cmd --query-port="{1}/{0}"'
+        exists_command = 'firewall-cmd --permanent --query-port="{1}/{0}"'
```

Comment:
--permanent makes the change persisted for when system reboots (but not change immediatelly) if need runtime change make another call without permanent as said in firewalld docs:


```
--permanent
The permanent option --permanent can be used to set options permanently. 
These changes are not effective immediately, only after service restart/reload or
 system reboot. Without the --permanent option, a change will only be part of the 
runtime configuration.

If you want to make a change in runtime and permanent configuration, 
use the same call with and without the --permanent option.

The --permanent option can be optionally added to all options further down 
where it is supported
```